wake-lock/wakelock-object-is-independent.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>WakeLock object is independent</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<script>
+
+function make_script() {
```

Comment:
I'd suggest putting this into an innert `<script type="plain/text">` or make this an ES6 template. You can even have this as a plain function, and then just call `f.toString()`, and it will give you the source.